tests/test__ped_partition.py

```diff
@@ -177,18 +182,39 @@ def runTest(self):
         self._part.set_name("blah")
         self.assertEqual(self._part.get_name(), "blah")
 
-@unittest.skip("Unimplemented test case.")
-class PartitionIsBusyTestCase(unittest.TestCase):
+class PartitionIsBusyTestCase(RequiresPartition):
+    def setUp(self):
+        RequiresPartition.setUp(self)
+        self.mountpoint = None
+
+    def doMount(self):
+        self.mountpoint = tempfile.mkdtemp()
+        os.system("mount -o loop %s %s" % (self.path, self.mountpoint))
+
+    def mkfs(self):
+        os.system("/sbin/mkfs.ext2 -F -q %s" % self.path)
+
+    def tearDown(self):
+        if self.mountpoint:
+            os.system("umount %s" % self.mountpoint)
+            os.rmdir(self.mountpoint)
+        RequiresPartition.tearDown(self)
+
     def runTest(self):
-        # TODO
-        self.fail("Unimplemented test case.")
+        # partitions aren't busy until they're mounted.
+        self.assertFalse(self._part.is_busy())
+
+# TODO:  need to figure out how to make a loopback device look mounted to
+# libparted
+#        self.mkfs()
+#        self.doMount()
+#        self.assertTrue(self._part.is_busy())
+
 
 class PartitionGetPathTestCase(RequiresPartition):
     def runTest(self):
         self.assertNotEqual(self._part.get_path(), "")
 
-@unittest.skip("Unimplemented test case.")
-class PartitionStrTestCase(unittest.TestCase):
+class PartitionStrTestCase(RequiresPartition):
     def runTest(self):
-        # TODO
-        self.fail("Unimplemented test case.")
+        self.assertTrue(str(self._part).startswith("_ped.Partition instance"))
```

Comment:
Well the test is called PartitionStrTestCase and I've modelled it right after this existing code:

```
class FileSystemTypeStrTestCase(unittest.TestCase):
    def runTest(self):
        fstype = _ped.file_system_type_get("ext3")

        self.assertEqual(str(fstype), "_ped.FileSystemType instance --\n  name: ext3")
```

startswith() is used b/c str() conversion gives too much details and includes object addresses which are harder to replicate in the expected string in the test.

Let me know what is the intention here and I can update accordingly.